.madrun.js

```diff
@@ -102,3 +141,16 @@ module.exports = {
     'build:client:dev': () => run('6to5:client:dev'),
     'heroku-postbuild': () => run('6to5:client'),
 };
+
+// ---- This does not compile!
+// function dockerPush(type, version = 'latest') {
+//     return `docker push coderaiser/cloudcmd:${version}-${type}`;
+// }
+
+// module.exports = {
+//     'docker:push:arm32': () => dockerPush('arm32', version),
+//     'docker:push:arm32:latest': () => dockerPush('arm32'),
+//     'docker:push:arm64': () => dockerPush('arm64', version),
+//     'docker:push:arm64:latest': () => dockerPush('arm64'),
+// };
+// ----
```

Comment:
What purpose of this commented code?

Reply:
Those lines were for the use of a generic `dockerPush()` function to be utilized for all `architectures` and `versions`. But this did not compile for me. Did this compile for you when uncommenting these lines?

Reply:
Looks like you tried to overwrite `module.exports` in the bottom of a file. This removed `fix:lint` and all other statements, you should put it to the place where other exports located :)

Reply:
Makes total sense, just changed it and now works just fine.